nixos/modules/services/backup/borgbackup.nix

```
@@ -310,6 +315,23 @@ let
     '';
   };
 
+  mkCreateCommandImportTarDumpCommandAssertion = name: cfg: {
+    assertion = cfg.createCommand != "import-tar" || cfg.dumpCommand != null;
```

```suggestion
    assertion = cfg.createCommand == "import-tar" -> cfg.dumpCommand != null;
```
is logically equivalent but much more readable